### PR TITLE
Jansen karl atienza

### PR DESCRIPTION
**Pull request by jansenatienza**

### CODE REVIEW

File: utils/variables.js [hunk 1]
```diff
@@ -1879,8 +1879,8 @@ const IT_3104 = [
     name: "Atienza, Jansen Karl H.",
     activities: {
       website: {
-        link: "",
-        code: "",
+        link: "https://jansenatienza.github.io/jansen-atienza/",
```
Comment (owenfalculan) [inline utils/variables.js:1882]: css and images not working https://jansenatienza.github.io/jansen-atienza/

Comment (owenfalculan) [inline utils/variables.js:1882]: both links not working/found

commands in updating branch:
git add .
git commit -m "update links"
git push

```suggestion
        link: "https://jansenatienza.github.io/atienza-jansen/",
```

**Review by owenfalculan — CHANGES_REQUESTED** [review]

**Comment by jansenatienza** [comment]
link: "https://jansenatienza.github.io/atienza-jansen/",
        code: "https://github.com/jansenatienza/atienza-jansen.git",

File: utils/variables.js [hunk 2]
```diff
@@ -1879,8 +1879,8 @@ const IT_3104 = [
     name: "Atienza, Jansen Karl H.",
     activities: {
       website: {
-        link: "",
```
Comment (jansenatienza) [inline utils/variables.js:1882]: link: "https://jansenatienza.github.io/atienza-jansen/",
        code: "https://github.com/jansenatienza/atienza-jansen.git",

**Pull request closed without merging**